.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Default image contents: https://hub.docker.com/r/mlpack/jenkins-amd64-debian
```

Comment:
Maybe the mlpack/mlpack image would be the better one to use?
https://hub.docker.com/r/mlpack/mlpack

I guess one question is, is the goal to have a container with mlpack already built and available, so that the user can develop applications that use mlpack?  Or is the goal to have a container that has mlpack's dependencies, so that a user can develop mlpack itself and submit PRs to the project?

In either case, it's trivial to add a new Dockerfile to [here](https://github.com/mlpack/jenkins-conf/tree/master/Dockerfiles) that we can just use directly.

Reply:
Absolutely, piece of cake to add another image.  [Here's](https://github.com/mlpack/jenkins-conf/blob/master/Dockerfiles/mlpack-dockerhub/Dockerfile) the existing Dockerfile for mlpack/mlpack; do you want to adapt that?  Or I can take a shot at it if you like.  Once we have something ready I can push it to Dockerhub.

I think it might be a good idea to not install mlpack in whatever container we use; I'm imagining a situation where the user modifies some core mlpack code (in wherever the mlpack source code is checked out), but now that code differs from what's in `/usr/include/` (the installed version) and strange linking or behavior issues happen when trying to use mlpack.

Reply:
@coatless let me know what you think on this point, I'm happy to help with a new Dockerfile. :+1:

Reply:
@rcurtin I'll take care of it later today in a PR over in the `mlpack/jenkins-conf` repository.

Reply:
Let's take the discussion on the Universal base over to: https://github.com/mlpack/jenkins-conf/pull/30

Short gist: I need to lobotomize the container to build it.